### PR TITLE
Fix some tests c-1140

### PR DESCRIPTION
**Pull request by nicoback2**
### Description
fixes 9/19 test suites
### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

File: packages/web/__mocks__/hashids.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class Hashids {
+  static mockDecode = jest.fn()
+  static mockEncode = jest.fn()
+  decode = this.mockDecode
+  encode = this.mockEncode
+}
+
+module.exports = Hashids
```
Comment (nicoback2) [inline packages/web/__mocks__/hashids.js:8]: I don't think this worked lol but leaving here for reference

File: packages/web/src/utils/route/userRouteParser.test.js [hunk 2]
```diff
@@ -12,7 +26,7 @@ describe('parseUserRoute', () => {
   })
 
   it('can decode a hashed user id route', () => {
-    mockDecode.mockReturnValue([11845])
+    // Hashids.mockDecode.mockReturnValue([11845])
 
     const route = '/users/eP9k7'
```
Comment (nicoback2) [inline packages/web/src/utils/route/userRouteParser.test.js:31]: This test now does not mock the `hashids` module but it works. But we should def figure out how to mock modules used by common... this is as far as I got for now.

File: packages/web/src/utils/route/userRouteParser.test.js [hunk 3]
```diff
@@ -2,6 +2,21 @@ import { parseUserRoute } from './userRouteParser'
 
 // eslint-disable-next-line
 import { mockDecode } from '__mocks__/Hashids'
+jest.mock('@audius/common', () => {
```
Comment (nicoback2) [inline packages/web/src/utils/route/userRouteParser.test.js:5]: should prob move this to setup tests later

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/nkang--fix-user-route-parser-test

**Review by amendelsohn — APPROVED** [review]
Great progress! Thanks for working on this!

**Pull request merged**